_includes/pbc/all/install-features/202307.0/install-the-dynamic-store-prices-feature.md

```diff
@@ -0,0 +1,517 @@
+{% info_block warningBox %}
+
+Dynamic Multistore is currently running under an Early Access Release. Early Access Releases are subject to specific legal terms, they are unsupported and do not provide production-ready SLAs. They can also be deprecated without a General Availability Release. Nevertheless, we welcome feedback from early adopters on these cutting-edge, exploratory features.
+
+{% endinfo_block %}
+
+This document describes how to integrate the  Dynamic Store + Prices feature into a Spryker project.
+
+## Install feature core
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME | VERSION |
+| --- | --- |
+| Spryker Core | {{page.version}} |
+| Prices | {{page.version}} |
+
+### 1) Set up configuration
+
+{% info_block warningBox "Configuration stores.php" %}
+
+Since the dynamic store is now enabled, configuration for the store is stored in the database, making the file `config/Shared/stores.php` deprecated. 
+
+The default store configuration will now be imported using new data import modules such as `CurrencyDataImport`. These modules will populate the store configuration in the database.
+A new major module `Currency`, have been introduced and that is responsible for extending store data and configuring it in the database. You can find migration guide [here](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/update-the-currency-module.html)
```

Comment:
is there any particular reason to link to the upgrade guide here?

Reply:
Just to make a journey for the reader easier. They read the installation doc and that they need to do a few major migrations and just to save them a few clicks I've added a link here already